### DISCUSSION TITLE
翻译错误上报

### DISCUSSION THREAD

**Discussion by qwq321qwq** [LocalizeLimbusCompany/LocalizeLimbusCompany#328] [常规 | General]
<img width="983" alt="微信图片_20241229084810" src="https://github.com/user-attachments/assets/287675a2-ca66-425a-845e-42373a069d47" />

**Comment by Nephthys137** [comment]
请问你认为哪里有问题

**Reply by qwq321qwq** [reply]
俺寻思.jpg，看错了看成炅了orz
